lms/services/grouping.py

```diff
@@ -50,6 +50,8 @@ def upsert_with_parent(
         :param parent: Parent grouping for all upserted groups
         :param type_: Type of the groupings
         """
+        if not grouping_dicts:
```

Comment:
Not really necessary but we already know about this case so we might as well handle it.